viewing_party/main.py

```diff
@@ -0,0 +1,224 @@
+# FUNCTION  FOR TESTS 1, 2, 3,  4  WAVE 1 PASSED
+def create_movie(movie_title, genre, rating):
+    '''
+    Creates a dictionary new-movie with the inputs as values.
+    Input: three parameters two strings, one number.
+    Output : returns the dictionary new-movie.
```

Comment:
Great job practicing docstrings for all these methods! It wasn't a requirement but very cool that you took the time to do that!